core/cmd/wandb-core/main.go

```diff
@@ -33,6 +34,12 @@ func main() {
 
 	flag.Parse()
 
+	var shutdownOnParentExitEnabled bool
+	if *pid != 0 {
```

Comment:
I would do `ppid := os.Getppid()` instead. I sometimes start core on its own and connect to it with different processes.

Reply:
also, not nil, since it's a pointer?